transformers/file_utils.py

```diff
@@ -26,17 +26,25 @@
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 try:
-    import tensorflow as tf
-    assert hasattr(tf, '__version__') and int(tf.__version__[0]) >= 2
-    _tf_available = True  # pylint: disable=invalid-name
-    logger.info("TensorFlow version {} available.".format(tf.__version__))
+    if 'NO_TF' in os.environ and os.environ['NO_TF'].upper() in ('1', 'ON'):
```

Comment:
maybe add 'Y' and 'YES' in the options? (also usually we use `lower()` but that's nitpicking)

Reply:
there's now a `parse_flag_from_env` in tests/utils that we could move to `src` and use here.

Reply:
but yeah that's helpful, I've been `pip uninstall`ing tensorflow a lot :)